client/src/main/java/org/asynchttpclient/netty/future/StackTraceInspector.java

```diff
@@ -39,6 +41,23 @@ public static boolean recoverOnNettyDisconnectException(Throwable t) {
             || (t.getCause() != null && recoverOnConnectCloseException(t.getCause()));
   }
 
+  public static boolean recoverOnChunkedUploadFailed(Throwable t) {
+    if (t instanceof ChannelOutputShutdownException) {
+      try {
+        for (StackTraceElement element : t.getStackTrace()) {
+          String className = element.getClassName();
+          String methodName = element.getMethodName();
+          if (className.contains("ChunkedWriteHandler") && (methodName.equals("flush") || methodName.equals("write")))
+            System.exit(1);
```

Comment:
o_0

Reply:
lol... sorry

Reply:
anyway build will fail due to some issues:
> Dynamic Oracle JDK installations are currently unavailable due to a problem with our upstream provider. In the meantime, container-based builds come with JDKs preinstalled.